meilisearch-http/tests/auth/api_keys.rs

```diff
@@ -719,6 +725,7 @@ async fn patch_api_key_description() {
     // Add a description
     let content = json!({ "description": "Indexing API key" });
 
+    thread::sleep(time::Duration::new(1, 0));
```

Comment:
Why are we sleeping in all these tests if we do not check anything time-related after?

```suggestion
```